packages/medusa/src/services/product.ts

```diff
@@ -876,8 +785,8 @@ class ProductService extends BaseService {
     delete query.relations
 
     return {
-      query,
-      relations: rels,
+      query: query as FindWithoutRelationsOptions,
```

Comment:
comment: this [PR](https://github.com/medusajs/medusa/pull/1672/files) should fix your issues around that :)

Reply:
Gave it a shot, and when conflicts are resolved, it causes an error in the price list service. We could very well have an issue when we merge, but i'd rather we tackle them as they arise and perhaps in a different ticket/PR. Wdyt?

Reply:
Yeah let see that in another pr such as the one i mentioned